newIDE/app/src/MainFrame/EditorContainers/HomePage/ManageSection/index.js

```diff
@@ -66,6 +66,15 @@ const ManageSection = ({
     onOpenLoginDialog,
   } = authenticatedUser;
 
+  React.useEffect(
+    () => {
+      onRefreshGames();
+    },
+    // Refresh the games when the section is opened, useful when a game gets registered.
```

Comment:
This will trigger an API call each time someone opens the game section. Is there another way to accomplish the expected result?

Reply:
I think the only other "clean" way would be to have a gameContext again. Shared everywhere, so that when a game gets registered from a place in the app (export for instance) then we can call a "onGameRegistered" that will trigger a fetch again.
Here, because the list is fetched for this component only, it just doesn't gets updated if it is modified from somewhere else in the app

Reply:
I expected this answer but maybe you thought of something else 😅